asdf/treeutil.py

```diff
@@ -357,9 +348,6 @@ def _handle_immutable_sequence(node, json_id):
             # The derived class signature is different, so simply store the
             # list representing the contents. Currently this is primarly
             # intended to handle namedtuple and NamedTuple instances.
-            if not ignore_implicit_conversion:
```

Comment:
Hmm... perhaps we should raise an error and halt serialization in this case?  It seems hazardous to perform any kind of implicit conversion when writing a file.

Reply:
See 1bf58ce.